.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-1.22.19.cjs
```

Comment:
I don't think we should be checking this and the target yarn file into version control here